ERROR - 2023-10-18 14:02:30 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan-19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:02:30 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(14): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:02:58 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan-19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:02:58 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:03:08 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan-19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:03:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:09:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:09:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:09:43 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:09:43 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:10:54 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:10:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:11:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:11:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:16:02 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:16:02 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:16:06 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:16:06 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:26:20 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:26:20 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:39:32 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:39:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:39:35 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:39:35 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
ERROR - 2023-10-18 14:44:06 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'perpustakaan_19...', 3306, '', 0)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#9 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\perpustakaan-19220577\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perpustakaan-19220577\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perpustakaan-19220577\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perpustakaan-19220577\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perpustakaan-19220577\app\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
#7 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
#8 C:\xampp\htdocs\perpustakaan-19220577\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perpustakaan-19220577\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perpustakaan-19220577\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-18 14:44:06 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_penerbit`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\PenerbitController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\PenerbitController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PenerbitController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\perpustakaan-19220577\\public\\index.php')
